apps/storybook/src/stories/TextField/TextField.stories.tsx

```diff
@@ -22,7 +22,7 @@ export default {
   },
 
   argTypes: {
-    status: { type: 'select', options: ['normal', 'error', 'read-only'] },
+    status: { type: 'select', options: ['normal', 'error', 'warning', 'read-only'] },
```

Comment:
since <TextInputBase/> is used in all Fields, maybe we should add showcases in these too